Assets/Scripts/UI/ConnectAction.cs

```diff
@@ -38,25 +39,24 @@ public void ConnectToServer()
             string networkAddress = serverAddress.text;
             IPAddress parsedAddress;
             ushort networkPort;
-            try
-            {
-                // Parse parameters
-                IPAddress.TryParse(networkAddress, out parsedAddress);
-                ushort.TryParse(serverPort.text, out networkPort);
-
-                // Verify parameters
-                // TODO: write verify code
+            // Parse parameters
+            bool validIP = IPAddress.TryParse(networkAddress, out parsedAddress);
+            bool validPort = ushort.TryParse(serverPort.text, out networkPort);
 
+            // Verify parameters
+            // TODO: write verify code
```

Comment:
What would this verify code be?
Some sort of way of verifying the ip and port strings are well formed?
If so, I guess some googled regex patterns should do the trick.

Reply:
Yeah, I'll make an issue for this. Right now there is just a try parse but it should be improved in the future to give user feedback through things like text, UI elements, progress meter, and everything else we need. 

Should also try to detect timeout when trying to connect or give feedback on why a request is rejected. 

Very simple state machine just need to scope it out and write it down.

Reply:
https://github.com/nicholas-maltbie/PropHunt/issues/136 Issue created